google-map-marker.html

```diff
@@ -366,7 +366,7 @@
         subtree: true
       });
 
-      var content = this.innerHTML.trim();
+      var content = this.children[0];
```

Comment:
No please, don't abandon it, it's working fine and we need actions from within the infowindow
I am using a all the changes (in lines 41, 369, 384, plus new line 399) and it works in both scenarios:
- with data binding and actions (code) within the infowindow element
- and also working fine with just text as you suggested (<google-marker>text content</google-marker>)